cmd/livepeer_cli/wizard.go

```diff
@@ -45,3 +46,24 @@ func (w *wizard) readDefaultString(def string) string {
 	}
 	return def
 }
+
+// readInt reads a single line from stdin, trimming if from spaces, enforcing it
+// to parse into an integer.
+func (w *wizard) readInt() int {
+	for {
+		fmt.Printf("> ")
+		text, err := w.in.ReadString('\n')
+		if err != nil {
+			log.Crit("Failed to read user input", "err", err)
+		}
+		if text = strings.TrimSpace(text); text == "" {
+			continue
+		}
+		val, err := strconv.Atoi(strings.TrimSpace(text))
```

Comment:
why do you have to TrimSpace again?

Reply:
I think otherwise it errors out... Copied from puppeth